src/components/Concentrator/ConcentratorVaultTable.tsx

```diff
@@ -51,7 +51,7 @@ export const ConcentratorVaultTable: FC<ConcentratorVaultTableProps> = ({
   const supportedChain = availableChains?.[0]
 
   return (
-    <VaultTable label={`${label} Vaults`} earningEnabled={false}>
+    <VaultTable label={`${label} Vaults`} showEarningsColumn={false}>
```

Comment:
I didn't catch this the first time, but you should just make this an optional prop and you won't need to explicitly set it to false when the column is not desired:

```ts
showEarningsColumn?: boolean
```

```tsx
// with earnings column
<VaultTable label="Foo" showEarningsColumn />
// without earnings column
<VaultTable label="Bar" />
```